db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20141010131030) do
+ActiveRecord::Schema.define(version: 20141015152508) do
```

Comment:
Someone mind explaining to me what this file is for, and why now and again it auto-updates and gets included in a pull request?  I've seen some people even roll back the file.